README.md

```diff
@@ -28,6 +28,7 @@ scalastyle {
   failOnWarning = false
   verbose = false
   quiet = false
+  options {} // JavaForkOptions that determine fork options of the daemon worker(s)
```

Comment:
Should we maybe call it forkOptions to make it more specific?

Reply:
Makes sense. I'll change it to `forkOptions`.

Reply:
Done :)